src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -534,6 +553,20 @@ $DOCKERFILE_SIGIL
     )
   }
 
+  lazy val dependencyImageTagDef: Def.Initialize[Task[String]] = Def.task {
+    val imageTag = imageNameWithLabel(dependencyImageDir.value, dependencyImageName.value)
+    val logger = Keys.streams.value.log
+    logger.info(imageTag)
+    imageTag
+  }
+
+  lazy val mainImageTagDef: Def.Initialize[Task[String]] = Def.task {
+    val imageTag = imageNameWithLabel(mainImageDir.value, mainImageName.value)
+    val logger = Keys.streams.value.log
+    logger.info(imageTag)
```

Comment:
If you use the `show` command in the sbt console, you don't need this log. It literally does exactly this.